### PR TITLE
Fix custom migrations

### PR DESCRIPTION
**Pull request by dmitrykruglov**

### CODE REVIEW

File: tasks/exodus.rake [hunk 1]
```diff
@@ -61,29 +61,29 @@ namespace Exodus.configuration.rake_namespace.to_s + 'db' do
         end
 
         puts "No migrations to run have been found" if migrations.empty?
-        Exodus::sort_and_run_migrations('up', migrations, step)
```
Comment (yingnanliu) [inline tasks/exodus.rake]: So is `sort_and_run_migrations` used anywhere? If not, we should delete it .

File: tasks/exodus.rake [hunk 2]
```diff
@@ -61,29 +61,29 @@ namespace Exodus.configuration.rake_namespace.to_s + 'db' do
...
+        Exodus::run_custom_migrations('up', migrations, step)
```
Comment (yingnanliu) [inline tasks/exodus.rake]: Any specific reason that you use `::` instead of `.`?

File: tasks/exodus.rake (hunk 1 above)
Comment (dmitrykruglov) [inline tasks/exodus.rake]: Yes, it's still used for running regular migrations

File: tasks/exodus.rake (hunk 2 above)
Comment (dmitrykruglov) [inline tasks/exodus.rake]: Just beause we have :: everywhere else in this file :)

**Pull request merged**